cmake/f3dOptions.cmake

```diff
@@ -135,10 +138,15 @@ function(_parse_json_option _top_json)
          # Use default_value
          string(APPEND _options_struct "${_option_indent}  ${_option_actual_type} ${_member_name} = ${_option_default_value_start}${_option_default_value}${_option_default_value_end};\n")
          set(_optional_getter "")
+         list(APPEND _options_is_optional "if (name == \"${_option_name}\") return false")
+       list(APPEND _options_reset "if (name == \"${_option_name}\") opt.${_option_name} = ${_option_default_value_start}${_option_default_value}${_option_default_value_end}")
```

Comment:
please align with previous line

Reply:
` ${_option_default_value_start}${_option_default_value}${_option_default_value_end}")`

You can factorize this part with the `option_struct` part

Reply:
done